### DISCUSSION TITLE
classList binding with css module?

### DISCUSSION THREAD

**Discussion by shi-yan** [solidjs/solid#2011] [Q&A]
With css module, the class name becomes styles.ClassName, but this can't be used in a classList binding:

```javascript
<button
  classList={{styles.TestCSSClass: current() === 'foo'}}
  onClick={() => setCurrent('foo')}
>foo</button>
```

`styles.TestCSSClass` is not a legit name for the classList binding, what should I do?

**Comment by pattuzzoj** [comment]
classList={{ [styles.TestCSSClass]: current() }}

Test it.
